includes/class-sensei-admin.php

```diff
@@ -1326,19 +1326,23 @@ public function get_course_order() {
 	}
 
 	public function save_course_order( $order_string = '' ) {
+		global $wpdb;
 		$order = array();
 
 		$i = 1;
 		foreach ( explode( ',', $order_string ) as $course_id ) {
 			if ( $course_id ) {
-				$order[]     = $course_id;
-				$update_args = array(
-					'ID'         => absint( $course_id ),
-					'menu_order' => $i,
+				$order[] = $course_id;
+
+				// phpcs:disable WordPress.DB.DirectDatabaseQuery.DirectQuery, WordPress.DB.DirectDatabaseQuery.NoCaching -- Performance improvement.
+				$wpdb->query(
```

Comment:
Hmm so the performance improvement comes from switching `wp_update_post` with a direct query? How did we compare the two solutions?

Reply:
We saw the big improvement in Learnomattic. It can also be related to some customization there, but nothing that we could find easily.
Locally, it seemed almost the same to me with a few courses. ;)